Misc/NEWS.d/next/Library/2019-09-25-00-37-51.bpo-38267.X9Jb5V.rst

```diff
@@ -0,0 +1 @@
+Add *timeout* parameter to :meth:`asyncio.loop.shutdown_default_executor`.
```

Comment:
Maybe give it more color? It defaults to None, but when called from the Runner it is set to 5 minutes.

Reply:
done